apps/benchmark/src/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createDOMRenderer, RendererProvider } from '@griffel/react';
+import { App } from './App';
+
+const renderer = createDOMRenderer(document);
+
+ReactDOM.render(
+  <RendererProvider renderer={renderer}>
+    <App />
+  </RendererProvider>,
+  document.getElementById('root'),
+);
```

Comment:
```suggestion
ReactDOM.render(
  <App />,
  document.getElementById('root'),
);
```

Let's not inject Griffel there and build benchmark's styles with plain CSS to avoid any chance that it somehow affects results.

Reply:
Done in [1b8f8fc](https://github.com/microsoft/griffel/pull/132/commits/1b8f8fcf7738941f92d01831626fadccaaf16612)